src/cmr/sizing/core.clj

```diff
@@ -22,32 +22,56 @@
 ;;; Note that all internal operations for estimating size are performed
 ;;; assuming bytes as the units.
 
-(defn get-measurement
+(defn- get-measurement
   [variable]
   (let [dims (get-in variable [:umm :Dimensions])
         total-dimensionality (reduce * (map :Size dims))
         data-type (get-in variable [:umm :DataType])]
     (* total-dimensionality
        (util/data-type->bytes data-type))))
 
+(defn- get-avg-gran-size
+  [vars]
+  (let [value (get-in (first vars) [:umm :SizeEstimation :AvgCompressionRateASCII])]
+    (if (string? value)
+      (read-string value)
+      value)))
+
+(defn- get-avg-compression-rate
+  [vars]
+  (let [value (get-in (first vars) [:umm :SizeEstimation :AverageSizeOfGranulesSampled])]
```

Comment:
What are the units for the average granule size? It looks like we're assuming bytes based on the calculations, but I'm not positive.

Reply:
We should be doing everything in bytes, internally. The value that EDSC gives us is up to them; let me ping them and see what is easiest (I'm guessing MB, if they are taking this from Granule metadata ... )

Reply:
Actaully, we can ping @abbottry here :-)